.github/workflows/CI.yml

```diff
@@ -10,7 +10,7 @@ jobs:
       - name: Run TESTS
         run: bundle exec fastlane unit_test
       - name: Generate SonarCloud Friendly Coverage Report
-        run: ./xccov-to-sonarqube-generic.sh ~/Library/Developer/Xcode/DerivedData/*/Logs/Test/*.xcresult > sq-generic.xml
+        run: ./xccov-to-sonarqube-generic.sh "~/Library/Developer/Xcode/DerivedData/*/Logs/Test/*.xcresult" > "sq-generic.xml"
```

Comment:
The internet lied to you.  These aren't the issue.

Reply:
We've made an update to this path to point directly at the new location.  Unfortunately my `**/*.xcresult` did not work, so we still have to point at the correct directory.  Upon doing that, we now seem to be gather test coverage for all 226 tests in the run.

Unfortunately (again) we're still not getting the coverage all the way to Sonarcloud.  So we'll have to keep looking deeper.